frontend/src/components/teacher/view-sessions/useAssessmentDataQuery.ts

```diff
@@ -4,9 +4,29 @@ import { useQuery } from "@apollo/client";
 import { GET_TEST_SESSIONS_BY_TEACHER_ID } from "../../../APIClients/queries/TestSessionQueries";
 import type { TestSessionOverviewData } from "../../../APIClients/types/TestSessionClientTypes";
 import AuthContext from "../../../contexts/AuthContext";
+import type { TestSessionStatus } from "../../../types/TestSessionTypes";
 import { getSessionTargetDate } from "../../../utils/TestSessionUtils";
 
-const useAssessmentDataQuery = (limit?: number) => {
+export type FormattedAssessmentData = {
```

Comment:
could we combine this type with [this one](https://github.com/uwblueprint/jump-math/blob/ee6fc5d82cc085d3515069ccdd2141c4a720bf27/frontend/src/components/teacher/view-sessions/TestSessionListItem.tsx#L16)? so we don't have duplicate